src/shared/conts.ts

```diff
@@ -9,4 +9,8 @@ export const NETWORK_NAMES: Record<NetworkId, string> = {
   [NetworkId['arbitrum-sepolia']]: 'Arbitrum Sepolia',
   [NetworkId['op-mainnet']]: 'Optimism',
   [NetworkId['op-sepolia']]: 'Optimism Sepolia',
+  [NetworkId['polygon-pos-mainnet']]: 'PolygonPoS',
+  [NetworkId['polygon-pos-mumbai']]: 'PolygonPoS Mumbai',
```

Comment:
I believe this is another user facing string, is the PoS suffix essential cc @silasbw